### PR TITLE
Set In Game Time

### PR DESCRIPTION
**Pull request by ChTBoner**
Mostly implements In Game Time.
Also some changes made by rustfmt.

Some changes were just made for my convenience. Don't hesitate to remove them if you don't like it.

### CODE REVIEW

**Comment by dagit** [comment]
First off, I really appreciate the help with the project.

Second thing, and this is on me because I haven't written any contributor notes, could you make this PR against the `development` branch? It should be the same as the release branch but I'm trying to create a workflow where the CI automatically makes releases from the release branch and all development happens on the development branch.

Thanks!

File: src/autosplitters/supermetroid.rs [hunk 1]
```diff
@@ -2233,10 +2233,10 @@ impl SNESState {
     ) -> Result<SNESSummary, Box<dyn Error>> {
         let start_time = Instant::now();
         let snes_data = client.get_addresses(&[
-            (0xF5008B, 2),
-            (0xF5079B, 3),
-            (0xF50998, 1),
-            (0xF509A4, 61),
+            (0xF5008B, 2),  // Controller 1 Input
```
Comment (dagit) [inline src/autosplitters/supermetroid.rs:2236]: Thanks for documenting this. I really should have made a comment like this myself.

**Review by dagit — COMMENTED** [review]
All the individual changes look great to me except, how do you switch between using IGT and RTA? It seems like these just adds tracking for IGT but actually running against IGT in your splits is handled elsewhere?

**Comment by ChTBoner** [comment]
You don't need to switch between RTA and IGT. It keeps track of both at the same time.
Then it's on the renderer side that you need to switch between one or the other depending on what you want to display.

**Pull request closed without merging**